usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
@mkudlej if common tests will be directly in `usmqe_tests/api/tendrl`, would it be possible to launch just the common tests (and not recursively all other tests from `gluster` or `ceph` directory)?